dart/lib/src/sentry_client.dart

```diff
@@ -44,11 +44,8 @@ class SentryClient {
   /// Instantiates a client using [SentryOptions]
   SentryClient._(this._options)
       : _random = _options.sampleRate == null ? null : Random() {
-    _stackTraceFactory = SentryStackTraceFactory(_options);
-    _exceptionFactory = SentryExceptionFactory(
-      _options,
-      _stackTraceFactory,
-    );
+    _stackTraceFactory = _options.stackTraceFactory;
+    _exceptionFactory = _options.exceptionFactory;
```

Comment:
you can remove `_stackTraceFactory` and `_exceptionFactory` and read them directly from the `_options` right now, no need to have references around.

Reply:
Or make the properties a `get` pointing to `_options`